src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+.nav__ul {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  margin: 0;
+}
+
+.nav__item {
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-color;
+  line-height: 60px;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)